tests/test_api_analyses.py

```diff
@@ -0,0 +1,92 @@
+# pylint: disable=redefined-outer-name
+
+"""
+Tests in this file shall test analyses
+relying on access to vibium-cloud API
+Tests need to be run in order
+-p no:randomly
+"""
+from pathlib import Path
+
+import os
+import json
+import uuid
+import pandas as pd
+from requests import HTTPError
+import pytest
+from mvg import MVG
+
+VALID_TOKEN = os.environ["TEST_TOKEN"]
+
+# Test data and session setup
+REF_DB_PATH = Path.cwd() / "tests" / "test_data" / "mini_charlie"
+SOURCE_ID_WAVEFORM = uuid.uuid1().hex  # generate a unique source per testrun
+
+
+@pytest.fixture()
+def waveform_source_with_measurements(session, waveform_source):
```

Comment:
This function can also be moved to `conftest.py` and later reused when testing other features

Reply:
I had moved this fixture to conftest